uploadTest

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Hmm, as long as it doesn't change the base64encode of the file; that might make a reliable recording for the mock impossible.

I'll give it a shot since it's a bit cleaner this way.